procurement_manager/README.rst

```diff
@@ -0,0 +1,4 @@
+Procurement manager
+===================
+With this module, selecting procurements, you may check, run, and cancel
```

Comment:
`With this module, you can select multiple procurement orders and check, run or cancel them at once.`